README.md

```diff
@@ -676,6 +678,33 @@ As a convenience, the `id` and `itemToString` props which you pass to
 
 ## Event Handlers
 
+Downshift has a few events for which it provides implicit handlers. Several of
+these handlers call `event.preventDefault()`. Their additional functionality is
+described below.
+
+### default handlers
+
+* `ArrowDown`: moves the highlighted index down by 1. If this shift key is held
+  when this event fires, the highlighted index will jump down 5 indices instead of 1.
+  NOTE: if the current highlighed index is within the bottom 4 indices, the top-most
```

Comment:
This isn't quite accurate. For example, if you're on the second to last item and arrow down with the shift key you will be placed at the forth item. It's always a movement of 5 regardless of where you are in the last. You might say it's the equivalent of doing the same action 5 times.

Reply:
> For example, if you're on the second to last item and arrow down with the shift key you will be placed at the forth item. It's always a movement of 5 regardless of where you are in the last.

Hmm, this isn't the behavior that I observe. I get the same behavior that was the point of confusion in #336, namely, if **any** of the final 5 indices (which I actually _did_ mistype here when I wrote 4) are highlighted and the shift key is held when an ArrowDown event is fired, the top-most index is highlighted.

![](http://g.recordit.co/QqqBdOp85V.gif)

Reply:
That's odd... I could have sworn the logic what do as I suggested. Maybe I changed it and forgot? 🤔 Or maybe there's something weird with that example.

Reply:
Just checked. Looks like that's the logic alright. I'm ok with that. This is a rarely used feature anyway and I think both behaviors are just as reasonable as the other. This is fine! Thanks!